lib/cocoapods-core/source.rb

```diff
@@ -160,7 +160,7 @@ def versions(name)
       return nil unless specs_dir
       raise ArgumentError, 'No name' unless name
       pod_dir = pod_path(name)
-      return unless pod_dir.exist?
+      return nil unless pod_dir.exist?
```

Comment:
this is unnecessary -- `return` will implicitly return `nil`

Reply:
OK,but why the line 161 write `return nil unless specs_dir`?